openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1945,7 +1945,7 @@ The following additional Credential Issuer metadata parameters are defined for t
           * `locale`: OPTIONAL. String value that identifies language of this object represented as language tag values defined in BCP47 [@!RFC5646]. There MUST be only one object for each language identifier.
 * `order`: OPTIONAL. Array of the claim name values that lists them in the order they should be displayed by the Wallet.
 
-It is recommended to define an `@context` value to communicate additional information such as which claims are mandatory-to-be-issued, type of claim value (i.e., string, number, etc.), the display properties of a Credential, and the order of the claim values when displayed as in (#jwt-vc-json).
+It is recommended to define an `@context` value to communicate additional information such as which claims are mandatory-to-be-issued, type of claim value (i.e., string, number, etc.), the display properties of a Credential, and the order of the claim values when displayed.
```

Comment:
I don't understand the reason for this recommendation or I don't understand what this recommendation is trying to tell the reader to do; it says the purpose of defining a `@context` value is "to communicate additional information" but all the information listed can, as far as I can see, be communicated in the issuer metadata. Is it saying you should define which claims are mandatory both in an @context and in the issuer metadata?

I'm not really familiar with `@context`, does the example in this section follow that recommendation? It doesn't appear to as https://www.w3.org/2018/credentials/examples/v1 doesn't define any display properties I think.

Reply:
when @context is used, I would imagine that would take precedence over issuer metadata regarding type of claim value, mandatory or not, etc. not sure about display and order. but honestly, I am ok deleting it.

```suggestion
```